src/error_hints.jl

```diff
@@ -1,12 +1,21 @@
+MathTypes{T,C} = Union{AbstractRGB{T},TransparentRGB{C,T},AbstractGray{T},TransparentGray{C,T}}
+
 # provided by https://github.com/JuliaLang/julia/pull/35094
 function register_hints()
```

Comment:
This is likely to be an experimental feature: https://github.com/JuliaLang/julia/issues/35671

Reply:
https://github.com/JuliaLang/julia/pull/35679 was merged. So, I think it should be fixed. However, I don't know whether we can continue to maintain such experimental features. The nightly build is always experimental, though.
 cc @johnnychen94, @timholy

Reply:
#193